stubs/inertia-react/webpack.mix.js

```diff
@@ -0,0 +1,18 @@
+const mix = require("laravel-mix");
+
+mix.setPublicPath("./webroot")
```

Comment:
Let's be consistent and use single quotes here in whole file as I've used single quotes in [`webpack.mix.js` file](https://github.com/ishanvyas22/asset-mix/blob/master/stubs/inertia-vue/webpack.mix.js). After this change the test will pass and we can able to merge this in.

```suggestion
mix.setPublicPath('./webroot')
```

Reply:
Sorry about that my local prettier setup was acting up.